test/specs/parsing/xquery/DefaultFunctionNamespace.tests.ts

```diff
@@ -0,0 +1,38 @@
+import * as chai from 'chai';
+import * as slimdom from 'slimdom';
+
+import { evaluateXPath, evaluateXPathToBoolean, evaluateXPathToNumber } from 'fontoxpath';
+
+let documentNode;
+beforeEach(() => {
+	documentNode = new slimdom.Document();
+});
+
+describe('DefaultFunctionDeclaration', () => {
+	it('Can create a default namespace with PI', () => {
+		chai.assert.equal(
+			evaluateXPathToNumber(
+				'declare default function namespace "http://www.w3.org/2005/xpath-functions/math"; pi()',
+				documentNode,
+				undefined,
+				{},
+				{ language: evaluateXPath.XQUERY_3_1_LANGUAGE }
+			),
+			Math.PI
+		);
+	});
+	it('Can create a default namespace and a function', () => {
+		chai.assert.equal(
```

Comment:
chai does have an `isTrue` function which we usually use instead of equaling to true.
https://www.chaijs.com/api/assert/#method_istrue